lib/discover.js

```diff
@@ -81,17 +83,16 @@ TesselSeeker.prototype.start = function(opts) {
           debug('Fetching name:', tessel.connection.host);
           tessel.getName()
             .then(function() {
+              self.usbScanCount++;
               self.emit('tessel', tessel);
               return Promise.resolve();
-            })
-            .catch(function(err) {
-              logs.err('unable to fetch name:', err);
-              return Promise.resolve();
             });
         }
-      }).catch(function() {
+      })
+      .catch(function() {
         return Promise.resolve();
```

Comment:
Why not check the connection time and report the booting Tessel here rather than preserving a count of open scans?